lib/index-api.js

```diff
@@ -33,6 +33,16 @@ module.exports = function(config, auth, storage) {
   app.use(expressJson5({ strict: false, limit: config.max_body_size || '10mb' }))
   app.use(Middleware.anti_loop(config))
 
+  // encode / in a scoped package name to be matched as a single parameter in routes
+  // TODO: routes should be modified to properly handle scopes, this is just a workaround
+  app.use(function(req, res, next) {
+    if (req.url.indexOf('@') != -1) {
+      // e.g.: /@org/pkg/1.2.3 -> /@org%2Fpkg/1.2.3, /@org%2Fpkg/1.2.3 -> /@org%2Fpkg/1.2.3
+      req.url = req.url.replace(/^(\/@[^\/%]+)[\/$]/, '$1%2F')
```

Comment:
In a character group `$` means `$`, you can try the code snippet above for yourself.

I guess the regexp should look like this:

```js
req.url = req.url.replace(/^(\/@[^\/%]+)\/(?!$)/, '$1%2F')
```

So trailing slash (`/@scope/`) won't get encoded accidentally.